firefox-ios/firefox-ios-tests/Tests/ClientTests/Mocks/MockShareTab.swift

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+@testable import Client
+
+class MockShareTab: ShareTab {
+    var displayTitle: String
+    var url: URL?
+    var webView: Client.TabWebView?
```

Comment:
nit; I think that's not needed
```suggestion
    var webView: TabWebView?
```

Reply:
Thanks, good catch! 😁